brax/physics/config.proto

```diff
@@ -61,6 +61,12 @@ message Collider {
     float size = 1;
     repeated float data = 2;
   }
+  // High-level message for containing elasticity and friction information
+  message Material {
+    // How much surfaces in contact resist translation
+    optional float friction = 1;  // When unspecified, defaults to global friction coefficient
```

Comment:
Let's remove the optional here, as the whole Material is already optional as a Message field.